### PR TITLE
Api config

### PR DESCRIPTION
**Pull request by hernan-clich**
## Summary

- Added .env environment variables config.
- Added api config.
- Created Book service with mocked data and implemented with redux action creator.
- Updated Reactotron config to include the proper `apisauce` plugin.

## Screenshots

No noticeable changes in the UI.

### Environment variables are recognized and printable on Reactotron:

rn_env_config

## Trello Card

https://trello.com/c/w3BBB5BR/71-n-94-api-ii-variables-de-ambiente
https://trello.com/c/rMouSogo/27-n-95-api-iii-servicios

### CODE REVIEW

File: wbooks/android/app/build.gradle [hunk 1]
```diff
@@ -185,6 +186,8 @@ dependencies {
     //noinspection GradleDynamicVersion
     implementation "com.facebook.react:react-native:+"  // From node_modules
 
+    implementation project(':react-native-config')
+
```
Comment (sfernandez11) [inline wbooks/android/app/build.gradle]: Not necessary, RN > 0.60 has autolink

File: wbooks/android/settings.gradle [hunk 2]
```diff
@@ -1,3 +1,5 @@
 rootProject.name = 'wbooks'
 apply from: file("../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesSettingsGradle(settings)
 include ':app'
+include ':react-native-config'
+project(':react-native-config').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-config/android')
```
Comment (sfernandez11) [inline wbooks/android/settings.gradle]: Same

File: wbooks/src/config/api.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { create } from 'apisauce';
```
Comment (sfernandez11) [inline wbooks/src/config/api.ts:1]: Did you install this dependency?

File: wbooks/src/services/BookService.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import { BOOKS_MOCK } from '@constants/mockBooks';
+import { BookServiceMockedResponse } from '@interfaces/bookServiceMock';
+
+const BookService = {
+  getBooks: (): Promise<BookServiceMockedResponse> =>
+    new Promise(resolve => {
+      const data = { ok: true, data: BOOKS_MOCK, problem: 'Something went wrong' };
```
Comment (sfernandez11) [inline wbooks/src/services/BookService.ts]: If `ok` === `true`, why the `data` has a `problem`? The problem is only when `ok: false`

**Review by sfernandez11 — CHANGES_REQUESTED** [review]
Great, some comments...

File: wbooks/src/services/BookService.ts (hunk 4 above)
Comment (hernan-clich) [reply wbooks/src/services/BookService.ts]: Added a reject case to handle that scenario.

**Review by sfernandez11 — APPROVED** [review]
🥇

**Pull request merged**